frontend/packages/kubevirt-plugin/src/utils/redux/common.tsx

```diff
@@ -4,7 +4,7 @@ let lastID = 0;
 
 const getNextReduxID = () => String(++lastID);
 
-export const withReduxID = <T, _>(Component: React.ComponentType<T>) => {
+export const withReduxID = <T,>(Component: React.ComponentType<T>) => {
```

Comment:
Using `<T,>` is better than `<T, _>` since the latter introduces an unused type parameter.

https://dev.to/tlylt/typescript-generic-function-reported-as-jsx-error-57nf